### PR TITLE
Create About page to explain each of the project's role in the DeepCell ecosystem.

### PR DESCRIPTION
**Pull request by willgraf**
Add an SVG diagram in `/public/images` to show on the /about page, and moved the contents of `repo_explanation.md` into accordian-style drop downs with the repository name as a header.

Also:

- Updated `@material-ui/core` and `@material-ui/icons` to fix some deprecation warnings.
- Added `@svgr/webpack` to enable importing SVG files as React components.
- Rebuilt `yarn.lock` to address CVE-2020-7720

---

[A demo is live!](https://deepcell.org/about)

### CODE REVIEW

File: src/About/About.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+class About extends React.Component {
+
+  render() {
+    const { classes } = this.props;
+
+    return(
+      <div className={classes.root}>
+        <div className={classes.heroContent}>
+          <Typography variant="h1" align="center" color="textPrimary" gutterBottom>
+            About DeepCell
```
Comment (MekWarrior) [inline src/About/About.js:48]: Can we remove this header? It is a bit redundant and doesn't fit with the aesthetic of the rest of the page.

Comment (willgraf) [reply src/About/About.js:48]: Sure, I've mocked that up (haven't pushed the commit yet). How does it look?

**Comment by MekWarrior** [comment]
I think a brief description of the DeepCell ecosystem just below the figure (like a caption) would help orient the user. We can workshop what the text should be, but maybe something like the following would be appropriate: 

DeepCell is a collection of resources created by the Van Valen lab to facilitate the development and application of new deep learning methods to biology. The DeepCell ecosystem seeks to address 3 interdependent needs: (1) Data Annotation and Management, (2) Model Development, and (3) Deployment and Inference.

File: src/About/About.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel-deployment-1-content"
+                id="panel-deployment-1-header"
+              >
+                <Typography className={classes.heading}>deepcell.org</Typography>
+                {/* <Typography className={classes.secondaryHeading}>I am an accordion</Typography> */}
+              </AccordionSummary>
+              <AccordionDetails>
+                <Typography component={'span'}>
+                  The deepcell.org websites provides a web interface to access trained deep learning models.
```
Comment (MekWarrior) [inline src/About/About.js:139]: Do we want to point to DeepCell.org as a place that people should depend on to get access to models? Or should it stay a prototype/proof-of-concept?

**Comment by ngreenwald** [comment]
This looks great! Let me know once you guys have looked through the text and I'll make another pass through and open a PR for more cleanup

**Comment by willgraf** [comment]
@ngreenwald I'd like to close #122 with this PR. Is there any other content you'd like to see in the model information?

**Comment by ngreenwald** [comment]
I think this is a great MVP. It would be nice if there was some way to have the description of required input data for each model more easily visible from the model dropdown section on the website. 

Right now, if someone goes to deepcell.org/predict, it wouldn't be obvious how to find out what the requirements are for each model, or even what the different models do.

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**